src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,175 @@
+<!--
+layout: post
+title: Configuring Babel 6 for Node.js
+date: 2015-12-31T01:23:42.811Z
+comments: true
+published: true
+keywords: JavaScript, Node.js, ES2015, Babel, transpiler
+description: Tutorial on configuring Babel for node.js to get up and running quickly
+categories: ES2015, Babel, node.js
+authorName: Hannan Ali
+authorLink: https://abdulhannanali.github.io
+authorPicture: //s.gravatar.com/avatar/89e5f7614cb88cd573359a953a09aa6e?s=80
+-->
+Hi! If you are like me, you are tired of writing the same old ES5 JS Code in your node applications.
+
+If yes, you can use the newer features of JavaScript ES2015 and ES2016 standards in your node applications today. ES2015 and ES2016 make the JavaScript development a cool breeze but hey, not every ES2015 feature is supported in our beloved [Node.js](https://nodejs.org).
+
+This is where [Babel](https://babeljs.io) comes to the rescue. Babel is a transpiler for JavaScript which transpiles your ES2015 and ES2016 code into ES5 and even ES3 code. In simple words it converts it into JavaScript that node.js can run and make you really happy.
+<!--more-->
+
+**Little Notice:** If you just want ES2015 features and don't want to Babelify stuff. You can use `--harmony` flag before running your node application. In order to access more harmony flags for staging and experimental features run this command `node --v8-options | grep harmony` but there aren't even all the features present in Node. So you may continue if you can access ES2015 but also need to access **ES2016** features.
```

Comment:
@UltCombo Last revision is fine, what about just removing the word dislike and replace it with some thing more generic such as don't want to cos in some cases this is not the matter of liking or disliking. I wrote it but I am asking for suggestion. Also thanks for correcting and updating this :)